package.json

```diff
@@ -9,13 +9,15 @@
     "packages/ethereum",
     "packages/utils",
     "packages/hoprd",
+    "packages/wasm",
```

Comment:
Trying to contain all wasm modules in a single node package which mess up the dependency graph badly since we will end up having most other packages depend on this. Moreover, why would we want to expose that logic is implemented in wasm to the outside?

Reply:
Yes I agree, but the idea is to have Rust WASM crates in each package (core, core-ethereum,...) and have that same Makefile and structure there. `wasm` is an additional package that might contain stuff that is not suitable to be placed in other existing packages - e.g. REAL.
But I'm happy to call it anything else than `wasm`.

Reply:
I understand, then the naming is just unfortunate. Since you use the version as an example shouldn't this be moved into `hoprd` directly?

Reply:
Could be eventually.